### DISCUSSION TITLE
Reset index

### DISCUSSION THREAD

**Discussion by akshaykasar** [mikel-brostrom/yolo_tracking#1064] [Q&A] [answered]
How can I reset the index of the tracking objects? Is there any function? or I need to take new instance of tracker object?

**Comment by mikel-brostrom** [comment, marked as answer]
Yup. You need a new tracker instance

**Reply by akshaykasar** [reply]
Hi Mike, Thanks for clarification. In future will there be any method to reset the index of tracker?

**Reply by mikel-brostrom** [reply]
Which tracker are you using?

**Reply by akshaykasar** [reply]
Deepsort or strongsort

**Reply by mikel-brostrom** [reply]
For strongsort you could add the following to `track.py`

`yolo.predictor.trackers[0].tracker._next_id = 1`

**Reply by mikel-brostrom** [reply]
Something similar is not possible for deepocsort at the moment
